02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,82 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Akshaya's Mini Bio</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+<h1 align="Center">Akshaya Saraf</h1>
+<h5 align="Center" ><i>13224 Appaloosa Chase Drive,Austin, TX - 78732 </i><hr> Now you know where to find me! </hr></h5>
+<div> <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/AAEAAQAAAAAAAAawAAAAJDU5YzhlODA2LTg5YjktNGE1Yy05MDMyLTc4Yjc3MTE1ZDRjZA.jpg" alt="Akshaya's mini Bio" class="Center"> </div>
+<h5 align="Center">Bio</h5>
+<p align= "Center">
+<pre>
+I'm a Bachelor of Industrial Engineering from BMS College of Engineering, and am currently pursuing a Master of Science degree in Engineering management.
+I'm learning how to code so that I can make cool websites like this one. I'm from Bangalore, India.
+and NOW I dont know what to say. I think coding is cool. Well I'm gonna just write some interesting facts for you:
+<ol>
+<li>There are more life forms living on your skin than there are people on the planet.</li>
+<li>Otters sleep holding hands.</li>
+<li>The three wealthiest families in the world have more assets than the combined wealth of the 48 poorest nations.</li>
+<li>A traffic JAM lasted for more than 10 days, with cars moving 0.6 miles a day.</li>
+<li>A human will eat on average 70 assorted insects and 10 spiders while sleeping.</li></ol>
+</pre></p>
 
+<table align="Center" style="width:70%">
+  <caption>A little bit ABOUT ME:</caption>
+  <tr>
+    <th>Hobbies</th>
+    <th>All time Fav.Movies</th>
+    <th>Fav.Songs</th>
+  </tr>
+  <tr>
+    <td>Photography</td>
+    <td>Baby's Day Out</td>
+    <td>Alive- Sia</td>
+  </tr>
+  <tr>
+    <td>Skiing</td>
+    <td>Home Alone</td>
+    <td>Me too-Megan</td>
+  </tr>
+  <tr>
+    <td>Kayaking</td>
+    <td>P.S.I love you!</td>
+    <td>Highway to hell (you have to know the artist!)</td>
+  </tr>
+  <tr>
+    <td>and now .....CODING!</td>
+    <td>Lion King</td>
+    <td>Piano - Billy Joel</td>
+  </tr>
+</table>
+<hr></hr>
+<h5 align="Center">TV Shows I like Currently:</h5>
+<table style="width:100%">
+ 
+  <tr>
+    <td><img src="https://media.giphy.com/media/N6kLxgegOoesw/giphy-tumblr.gif"  style="width:500px; height:300px"/></td>
+    <td><img src="https://media.giphy.com/media/3oEjI0S99KxsaKmgX6/giphy.gif" style="width:500px; height:300px"/></td>
+
+  </tr>
+  <tr>
+    <td>FRIENDS</td>
+    <td>GAME OF THRONES!</td>
+  </tr>
+  <tr>
+       <td><img src="https://media.giphy.com/media/MzIGJbgcDFLgY/giphy.gif" style="width:500px; height:500px"></td>
+      <td><img src="https://media.giphy.com/media/5HLpEK5qSuXzG/giphy.gif" style="width:600px; height:500px"/></td>
+    </tr>
+  <tr>
+    <td>How I met your Mother..</td>
+    <td>Suits!</td>
+  </tr>
+</table>
```

Comment:
fantastic use of tables and background image! keep stretching your limits with code and trying new things.